src/pages/ProblemEditor/ProblemEditorContainer/ProblemEditorContainer.tsx

```diff
@@ -34,7 +37,7 @@ export const ProblemEditorContainer = () => {
   const formRef = useRef<FormikValues>();
 
   return (
-    <Box display="flex" flexDirection="column" flexGrow={1}>
+    <Box display="flex" flexDirection="column" flexGrow={1} textAlign="left">
```

Comment:
does this work with RTL?

Reply:
I'm not sure specifically what you are asking about